website/src/pages/index.module.css

```
@@ -8,6 +8,10 @@
 	display: none;
 }
 
+.HeroChartContainer * {
```

```suggestion
.HeroChartContainer table
```

shouldn't we do this instead? 🤔 Or there are another nodes where docusaurus' overflow rules are applied?